apps/web/src/components/HyloCookieConsent/HyloCookieConsent.js

```diff
@@ -9,6 +9,19 @@ export default function HyloCookieConsent () {
   const [cookieConsent, setCookieConsent] = useState(false)
   const [showCookieInfo, setShowCookieInfo] = useState(false)
 
+  useEffect(() => {
```

Comment:
Oh nice... I'm thinking about trying to squeeze in GDPR finishes

Reply:
Hop on in, the water is lukewarm